### ISSUE TITLE
[rule: no-unused-vars] not working properly

### ISSUE THREAD

**Issue by anshulguleria** [babel/babel-eslint#208]
I am using `eslint --version   #v1.9.0`.
 My sample code is:
```js
var test = function (name, pwd) {
    return {
        name: "some name"
    };
}
test();
```
This snippet should give error for `name` argument not getting used but it is not giving error, but its giving proper error for `pwd` argument.

**Comment by anshulguleria** [comment]
eslint/eslint#4446

**Comment by anshulguleria** [comment]
eslint#4446

**Comment by anshulguleria** [comment]
I think its an issue in eslint. Reporting there eslint/eslint#4446

**Comment by hzoo** [comment]
Closing then
